src/main/resources/application.yml

```diff
@@ -43,4 +45,4 @@ delius:
     username: DefaultValue
 
 azure.application-insights.web.enable-W3C: true
-azure.application-insights.web.enable-W3C-backcompat-mode: true
\ No newline at end of file
+azure.application-insights.web.enable-W3C-backcompat-mode: true
```

Comment:
better to define as yml i.e.:```
azure:
  application-insights:
    web:
      enable-W3C: true```, don't think you need backwards compatibility?

Reply:
Its a change Simon just put in - happy to remove if its not required?

Reply:
sorry, I saw the line had been modified so thought you had changed it, please ignore